server/protocols/jmap/src/main/java/org/apache/james/jmap/mailet/VacationReply.java

```diff
@@ -69,6 +76,9 @@ public VacationReply build() throws MessagingException {
 
         private MimeMessage generateMimeMessage() throws MessagingException {
             MimeMessage reply = (MimeMessage) originalMail.getMessage().reply(NOT_REPLY_TO_ALL);
+            if (subject.isPresent()) {
+                reply.setHeader("subject", subject.get());
```

Comment:
setHeader throws MessagingException

This is the more elegant way to deal both with exception handling, and code beauty ;-)